aws/table_aws_iam_policy.go

```diff
@@ -233,6 +240,37 @@ func getPolicyVersion(ctx context.Context, d *plugin.QueryData, h *plugin.Hydrat
 	return version, nil
 }
 
+func getPolicyUsage(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+	plugin.Logger(ctx).Trace("getPolicyUsage")
+	policy := h.Item.(*iam.Policy)
+
+	// Create Session
+	svc, err := IAMService(ctx, d)
+	if err != nil {
+		return nil, err
+	}
+
+	params := &iam.ListEntitiesForPolicyInput{
+		PolicyArn: policy.Arn,
+	}
+
+	policyUsageData := &iam.ListEntitiesForPolicyOutput{}
+
+	err = svc.ListEntitiesForPolicyPages(
+		params,
+		func(page *iam.ListEntitiesForPolicyOutput, lastPage bool) bool {
```

Comment:
@cjsrkd3321 as I can see you have first updated it as below and I believe that was the correct approach as with the current update, we are not using pagination.
```
policyUsageData := []*iam.ListEntitiesForPolicyOutput{}

	err = svc.ListEntitiesForPolicyPages(
		params,
		func(page *iam.ListEntitiesForPolicyOutput, lastPage bool) bool {
			policyUsageData = append(policyUsageData, page)
			return !lastPage
		},
	)
	
```

Reply:
@cjsrkd3321 Thanks for the response 👍 , I guess, I was not very clear in the above note. This is what we want you to consider again 
Your first [fix](https://github.com/turbot/steampipe-plugin-aws/pull/815/commits/a2372936fe1a4fdfbf71f53fb0594a4d7ecd3c17) after my suggestion was good

However, you changed the same in the [latest](https://github.com/turbot/steampipe-plugin-aws/pull/815/commits/5fcbdeb403abf3c1408b204fe5b5f001d336533e)

Pls do as below to retain your first change

```
func getPolicyUsage(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
	plugin.Logger(ctx).Trace("getPolicyUsage")
	policy := h.Item.(*iam.Policy)

	// Create Session
	svc, err := IAMService(ctx, d)
	if err != nil {
		return nil, err
	}

	params := &iam.ListEntitiesForPolicyInput{
		PolicyArn: policy.Arn,
	}
	policyUsageData := []*iam.ListEntitiesForPolicyOutput{}

	err = svc.ListEntitiesForPolicyPages(
		params,
		func(page *iam.ListEntitiesForPolicyOutput, lastPage bool) bool {
			policyUsageData = append(policyUsageData, page)
			return !lastPage
		},
	)

	if err != nil {
		return nil, err
	}

	return policyUsageData, nil
}
```
Pls revert back if you need further help

Reply:
I finished revert and re-commit. At first fix's output is [{"PolicyGroups": [{"GroupName": "BLAH"}]}] but, actually output is [this](https://docs.aws.amazon.com/cli/latest/reference/iam/list-entities-for-policy.html#examples). therefore, I just modified code. anyway i respect your opinion thank you!